docs/tutorials/mobile/pose-detection.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Pose detection with Yolov8
+description: Run the Yolov8 model with built-in pre and post processing
+image: /images/tutorial-superres-og-image.png
+parent: Deploy on mobile
+grand_parent: Tutorials
+nav_order: 3
+---
+
+# Pose detection with the Yolov8 model
+
+Learn how to build an ONNX model for pose detection with built-in pre and post processing.
+
+Note: this tutorial uses Python. Android and iOS samples are coming soon!
+
+## Build the Yolov8 model with pre and post processing
+
+Create a Python environment and install the following packages.
+
+```bash
+onnx
+onnxruntime
+onnxruntime-extensions
+```
+
+Download the following script to build the model.
+
+```bash
+https://raw.githubusercontent.com/microsoft/onnxruntime-extensions/main/tutorials/yolov8_pose_e2e.py > yolov8_pose_e2e.py
+```
+
+Run the script.
+
+```bash
+python yolov8_pose_e2e.py 
+```
+
+After the script has run, you will see one PyTorch model and two ONNX models:
+* `yolov8n-pose.pt`: The original Yolov8 PyTorch model
+* `yolov8n-pose.onnx`: The exported Yolov8 ONNX model
+* `yolov8n-pose.with_pre_post_processing.onnx`: The ONNX model with pre and post processing included in the model
+
+
+## Run examples of pose detection
+
+You can use the same script to run the model, supplying your own image to detect poses.
+
+```bash
+python yolov8_pose_e2e.py --input_image person.jpg
+```
+
+And the output is drawn on the original image!
```

Comment:
In my test, it save a temp file and try to open UI to show it. But it failed since I run in azure VM:
```
xdg-open: no method available for opening '/tmp/tmpy9pmki5z.PNG'
```
Probably add a line to save the image to current folder before displaying.

Reply:
We can work on fixing that in the original script